### PR TITLE
add eslint and prettier

### PR DESCRIPTION
**Pull request by facalz**
🔧 chore(.prettierrc): add prettier configuration file
🔧 chore(package.json): add format and lint scripts, update devDependencies
🔧 chore(src/__puppeteer.d.ts): remove semicolons
🔧 chore(src/global.d.ts): remove semicolons

✨ feat(index.ts): add support for buying items with non-zero price
🐛 fix(index.ts): fix console.log statements to use template literals instead of concatenation
🚨 chore(index.ts): fix linting issues and formatting

🎨 style(types.d.ts): format code and remove semicolons

🐛 fix(actions.ts): remove unused import and fix syntax errors in buy function

🔨 refactor(preparedRequests.ts): change quotes to single quotes and remove semicolons
🐛 fix(preparedRequests.ts): fix typo in getItemDetails parameter name
✨ feat(preparedRequests.ts): add support for getting marketplace item details

🎨 style(roblox.ts, timers.ts): fix code formatting and remove unnecessary semicolons

🔧 chore(token.ts): refactor token.ts to use ES6 imports instead of require
🔧 chore(tsconfig.json): exclude node_modules from TypeScript compilation

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,14 +5,21 @@
     "cheerio": "^1.0.0-rc.12",
     "dotenv": "^16.0.3",
     "puppeteer": "^19.8.5",
-    "typescript": "^5.0.4"
+    "typescript": "*"
```
Comment (thurdev) [inline package.json:8]: Any reason?

File: .prettierrc [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "semi": false,
```
Comment (thurdev) [inline .prettierrc]: if its related to ```;``` set to true.

File: package.json (hunk 1 above)
Comment (facalz) [reply package.json:8]: this will always get the latest available version

File: .prettierrc (hunk 2 above)
Comment (facalz) [reply .prettierrc]: done [#315a864](https://github.com/thurdev/roblox-limiteds-sniper-free/pull/10/commits/315a864091b9c129e09b5f198a9feafd42d25d3a)

**Pull request merged**